### DISCUSSION TITLE
Adding a "Medium" style zoom on images

### DISCUSSION THREAD

**Discussion by markbattistella** [retypeapp/retype#315] [Ideas]
I know this probably doesn't suit everyone, but some docs I've written have images that are wider than they are tall.

When viewing in the site when the right sidebar is collapsed it is an okay viewing experience, but if the user wants to have that pane open to see the titles the image is too squished.

Would it be possible to add in something like a zoom style on the images, and anything classes as `no-zoom` doesn't get it.

It would also be something activated than automatically turned on, and if it could be even further extended added to the `index.yml` as a `image-zoom: false/true` so you can disable on certain pages.

This is one I was using from another document site generator, and I have edited the bottom. However, adding this into the `_includes` doesn't seem to find the images.

See: https://github.com/francoischalifour/medium-zoom

[zoom-image.txt](https://github.com/retypeapp/retype/files/8434946/zoom-image.txt)

**Comment by dkkent** [comment]
Great idea!

**Comment by didntchooseaname** [comment]
upvoted!
